### PR TITLE
Add option for adaptive learning rate

### PR DESCRIPTION
**Pull request by camillebrianceau**
Add an adaptive learning rate with the function `ReduceLROnPlateau(optimizer, mode="min", factor=0.1, verbose=True )` . I didn't add any option to change the factors, the mode or the patience (default is 10). 
The update of the learning rate is based on the validation loss. 

Don't know if I need to add all these options or not.

### CODE REVIEW

File: clinicadl/utils/metric_module.py [hunk 1]
```diff
@@ -92,7 +92,8 @@ def mse_fn(y, y_pred):
         Returns:
             (float) mean squared error
         """
-
+        print("le")
```
Comment (ncassereau) [inline clinicadl/utils/metric_module.py]: forgotten prints

**Review by ncassereau — APPROVED** [review]
